src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package task;
```

Comment:
Could this be packaged with the bot's main functions?

Reply:
Hi! Could I clarify what you mean by packaging with the bot's main functions? Thank you!